script/parse-examples

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+function checkout() {
+  repo=$1; url=$2; sha=$3
+
+  if [ ! -d "$repo" ]; then
+    git clone "https://github.com/$url" "$repo"
+  fi
+
+  pushd "$repo"
+  git fetch && git reset --hard "$sha"
+  popd
+}
+
+# checkout examples/elsa ucsd-progsys/elsa 00e0bb97326cce5329dee8652f1531a4ed111d19
```

Comment:
Not sure if this should be commented. It's the only use of the `checkout()` function I see, and my intuition says you intended the function to be used at least once.

Reply:
ah yes, I did this when testing because one of the files in the repo had a typo in it, specifically elsa/tests/further/fixpoint.lc 
```elsa
  <b= F ((\f -> (\x -> f (x x)) (\y -> f (y y))) F)
  ```
so I manually removed it to test
  
 Added to known_failures now!